src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -1,81 +1,87 @@
 import React, { useState, useEffect } from 'react';
+import { useOktaAuth } from '@okta/okta-react';
 import { Header } from '../../common';
 import { Row, Col, Button } from 'antd';
-import { QuestionCircleOutlined } from '@ant-design/icons';
 import { useHistory } from 'react-router-dom';
 import FaceoffContent from './FaceoffContent';
-import { InstructionsModal } from '../../common';
-import { modalInstructions } from '../../../utils/helpers';
+import { getGameVotes } from '../../../api';
 
 const RenderMatchUp = props => {
-  const { push } = useHistory();
   const [faceoffs, setFaceoffs] = useState([]);
-  const [modalVisible, setModalVisible] = useState(true);
-
+  const [numberOfTimesVoted, setNumberOfTimesVoted] = useState(4);
+  const { authState } = useOktaAuth();
+  const { push } = useHistory();
   useEffect(() => {
-    setFaceoffs(props.faceoffs);
+    setFaceoffs(props.squad);
   }, [props]);
-
   const handleVote = e => {
     e.preventDefault();
     push('/child/squad-vote');
   };
-
-  const back2Dash = e => {
+  const backButton = e => {
+    e.preventDefault();
     push('/child/dashboard');
   };
-
+  useEffect(() => {
+    getGameVotes(authState, props.squad[0].SquadID, props.child.memberId).then(
+      res => {
+        setNumberOfTimesVoted(res.length);
+      }
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
   return (
     <>
-      <Header
```

Comment:
why are we deleting this as well?

Reply:
I think the header is still there, just a few lines farther down?  I didn't try to delete any header...

Reply:
By the looks of the insertions and deletions the header looks completely deleted, but we'll figure it out